simpleLooping/removeSpaceAtBeginning.py

```diff
@@ -17,3 +17,11 @@
 #   removeSpaceAtBeginning("")                  # return ""
 #   removeSpaceAtBeginning("Hello  World!")     # return "Hello  World!"
 #   removeSpaceAtBeginning("  Hello  World!  ") # return "Hello  World!  "
+from typing import List
+
+def removeSpaceAtBeginning(s: str) -> str:
+    i = 0
+    while i < len(s) and s[i] == " ":
```

Comment:
Should use ‘ ‘ for a single character. Not “ “.

When reading code, we can recognize it right away that it is a single space if you use ‘ ‘. But you use “ “ make it hard to know that it is single space or double space just by look at it.

Reply:
ok